examples/ciao/ciao.yml

```diff
@@ -7,10 +7,6 @@
   roles:
```

Comment:
This one looks nice, but still **missing the URL** pointing to the proper documentation of all the variables